noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr

```diff
@@ -99,139 +73,33 @@ fn field_to_bignum(x: Field) -> F {
     BigNum { limbs: __x_limbs }
```

Comment:
I can't comment on the line I want to, but here's a brillig optimisation from a few weeks ago, for converting to bignum, which should hopefully result in a small speed improvement (vs the to/from bytes conversions that are currently happening):

https://github.com/iAmMichaelConnor/blob-lib/blob/82ab7d3e3f43c4e7ab440edcdb3bcfb670c1545c/noir-circuits/blob/src/main.nr#L85C1-L109C2

with test:

https://github.com/iAmMichaelConnor/blob-lib/blob/82ab7d3e3f43c4e7ab440edcdb3bcfb670c1545c/noir-circuits/blob/src/main.nr#L569C1-L577C2